### ISSUE TITLE
knip v3.0.1 throwing error on a fresh run

### ISSUE THREAD

**Issue by amaccann** [webpro/knip#367]
`knip` and `typescript` installed, getting this error. `rm -rf` my entire node modules directory, installed from fresh but still getting this error. removed v2 `knip` config, just in case.

```js
Connecting the dots...
file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/typescript/visitors/imports/importDeclaration.js:20
                    imports.push({ symbol, specifier, identifier: '*', pos: symbol.declarations[0].pos ?? node.pos });
                                                                                   ^

TypeError: Cannot read properties of undefined (reading 'declarations')
    at file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/typescript/visitors/imports/importDeclaration.js:20:84
    at file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/typescript/visitors/index.js:4:35
    at visit (file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/typescript/getImportsAndExports.js:134:33)
    at visitNodes (/Users/amaccann/projects/path/monorepo/node_modules/typescript/lib/typescript.js:28081:24)
    at forEachChildInSourceFile (/Users/amaccann/projects/path/monorepo/node_modules/typescript/lib/typescript.js:28706:18)
    at Object.forEachChild (/Users/amaccann/projects/path/monorepo/node_modules/typescript/lib/typescript.js:28166:37)
    at visit (file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/typescript/getImportsAndExports.js:162:12)
    at getImportsAndExports (file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/typescript/getImportsAndExports.js:164:5)
    at ProjectPrincipal.analyzeSourceFile (file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/ProjectPrincipal.js:117:47)
    at analyzeSourceFile (file:///Users/amaccann/projects/path/monorepo/node_modules/knip/dist/index.js:185:66)
```

**Comment by amaccann** [comment]
For detail: running `v18.16.0` of node

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.0.2. See [Release 3.0.2](https://github.com/webpro/knip/releases/tag/3.0.2) for release notes._

**Comment by webpro** [comment]
Thanks, Andrew!
